Cue/app/api/Library.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+import { serverURL } from '../env';
+
+
+var LibraryApi = {
+	//TODO: not sure proper way to get state in the api
```

Comment:
When we add more API calls in the future, we could look at adding some static properties to, say, `AuthenticationApi` to avoid needing to pass the entire app state into every API call.